api/users.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt')
+const { JWT_SECRET } = process.env;
+const { createUser, getUser, getUserByUsername } = require('../db');
+const { requireUser } = require('./utils');
+
+
+// users/
+router.get('/', async (req, res, next) => {
+    const users = await getUser();
+    try {
+        res.send({
+            users
+        })
+    } catch (error) {
+        next(error)
+    }
+});
+
+// users/my account
+router.get('/myaccount', requireUser, async (req, res, next) => {
+            res.send({
+                ...req.user
+            })
+})
```

Comment:
we have no way of setting req.user currently so this function isn't working. Usually we build a JWT authorization middleware in our /api/index.js to handle this for us. It's purpose is to check that the token sent in the header of the request is valid, and then if it is valid it will store the user object in req.user